phpunit.xml.dist

```diff
@@ -0,0 +1,17 @@
+<phpunit
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/9.2/phpunit.xsd"
+	bootstrap="tests/bootstrap.php"
+	backupGlobals="false"
+	colors="true"
+	convertErrorsToExceptions="true"
+	convertNoticesToExceptions="true"
+	convertWarningsToExceptions="true"
+	convertDeprecationsToExceptions="true"
+	>
```

Comment:
I suggest adding this also:
```
cacheResultFile=".cache/.phpunit.result.cache"
```

And add the .cache dir to .gitignore.  It's inconvenient in PhpStorm when the phpunit cache is in the root, because I can only exclude directories from indexing, not individual files.

Reply:
Added in [29fd30c](https://github.com/aspirepress/AspireUpdate/pull/87/commits/29fd30cd70827a7b0046a31171943518ca790894).